pkg/certificates/cacertificate.go

```diff
@@ -96,7 +96,7 @@ func caCertExistsInSecret(secret *corev1.Secret) bool {
 	return certOk && privKeyOk
 }
 
-func generateCaCerttificate() ([]byte, []byte, error) {
+func generateCaCertificate() ([]byte, []byte, error) {
 	caCertTemplate, caPrivKey, err := generateCaCerttificateTemplate()
```

Comment:
I know its not your mistake.But if you are at it, you can correct this spelling also.